set.mm

```diff
@@ -7742,13 +7742,6 @@ inference eliminates the need for a syllogism ( ~ syl ) -type inference
       -> ( ( ( ph \/ ps ) \/ ch ) -> ( ph \/ ch ) ) ) $=
     ( wi wo orel2 ax-1 ja orim1d ) BADABEZACBAJADBAFAJGHI $.
 
-$(
--.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
-  Assorted results not assigned yet
--.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
-
-$)
-
```

Comment:
There should be a section header for AND - OR mixed theorems, shouldn't it?

Reply:
Yes.  @benjub wanted to improve on the one  of the previous section, so I wait until he is ready.

Reply:
Yes, let's do the sectioning and titles after the reordering is done. Not much left to do, as I understand, @wlammen ?

Reply:
Not really.  This is on the level of janitorial work.

Reply:
Opened Issue #2751 for that.